polkadot-wiki/sidebars.js

```diff
@@ -66,31 +68,35 @@ module.exports = {
             },
             {
               type: "category",
-              label: "Components",
+              label: "Staking",
               items: [
-                "learn/learn-architecture",
-                "learn/learn-collator",
-                "learn/learn-consensus",
-                "learn/learn-governance",
-                "learn/learn-gov2",
-                "learn/learn-identity",
+                "learn/learn-staking",
+                "learn/learn-nomination-pools",
+                "learn/learn-staking-faq",
                 "learn/learn-nominator",
```

Comment:
The nominator page contains some advanced topics such as bags list. This topic is already explained in the advanced staking page. I see the basics page as something containing really easy info that all user can understand if the want use polkadot. probably everything related to the UI must be put in advanced, and leave the dashboard-realted things in basics?